lib/rubocop/cop/naming/predicate_method.rb

```diff
@@ -156,7 +167,8 @@ def all_return_values_boolean?(return_values)
         end
 
         def boolean_return?(value)
-          value.boolean_type? || (value.call_type? && value.comparison_method?)
+          value.boolean_type? ||
+            (value.call_type? && (value.comparison_method? || value.predicate_method?))
```

Comment:
Can you refactor to the folliwng?

```suggestion
          return true if value.boolean_type?

          value.call_type? && (value.comparison_method? || value.predicate_method?)
```